packs/data/strength-of-thousands-bestiary.db/urbel.json

```diff
@@ -1000,8 +977,8 @@
                         "predicate": [
                             "action:shove"
                         ],
-                        "selector": "shove",
-                        "text": "A critical failure when attempting to Shove a creature with the weapon is treated as a failure instead.",
+                        "selector": "athletics",
+                        "text": "PF2E.NPCAbility.Urbel.ShoveNote",
```

Comment:
Why a note and not a degree-of-success adjustment?

Reply:
Because the ability says "... when attempting to Shove a creature with the weapon...". I don't think we can predicate this. If we're ok with assuming a shove will be with the weapon, a degree of success adjustment would be better.

Reply:
You should be able to predicate it

Reply:
Switched from Note to AdjustDegreeOfSuccess with a RollOption.